README.md

```diff
@@ -98,26 +97,48 @@ Releases will be (git tags and npm packages) will be signed either by [keybase/j
 To start using Taquito in your application, please visit [Taquito QuickStart](https://tezostaquito.io/docs/quick_start).
 Alternatively, get started with the [Taquito boilerplate project](https://github.com/ecadlabs/taquito-boilerplate#getting-started) 
 
-## Contributors Getting Started
+## Supported versions of Node.js®
 
-### NPM Scripts
+| Node.js version  |     |
```

Comment:
Could you please also add ES 5, not ES 6
It would be good to call it "Dependencies" or something like that more generically